CHANGELOG.asciidoc

```diff
@@ -3,6 +3,29 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.17.25]]
+=== Beats version 7.17.25
+https://github.com/elastic/beats/compare/v7.17.24\...v7.17.25[View commits]
+
+==== Breaking changes
+
+*Osquerybeat*
+
+- Disable `allow_unsafe` osquery configuration. {pull}40130[40130]
+
+==== Bugfixes
+
+*Metricbeat*
+
+- Add missing ECS Cloud fields in GCP `metrics` metricset when using `exclude_labels: true`. {issue}40437[40437] {pull}40467[40467]
+
+==== Added
+
+*Affecting all Beats*
+
+- Update Go version to 1.22.7. {pull}41019[41019]
```

Comment:
```suggestion
- Update Go version to 1.22.8. {pull}41106[41106]
```

https://github.com/elastic/beats/commits/ef6504bc5cb524dfe5000d367f8d775dc7e82473 is BC1's commits which includes the bump to 1.22.8.